sdk/consumption/Microsoft.Azure.Management.Consumption/tests/ScenarioTests/BudgetTests.cs

```diff
@@ -51,7 +51,7 @@ public void BudgetCreateOrUpdateTest()
         [Fact]
         public void BudgetDeleteTest()
         {
-            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            using (MockContext context = MockContext.Start(this.GetType().Name))
```

Comment:
If you are changing all of these anyway change them to pass only `this.GetType()` so `MockContext` can own naming decisions

Reply:
That seems like a reasonable idea but we would still need to have some mode that consumers can set to determine whether or not to use full or short name.

Reply:
Always use the short name and if two tests have the same name change it.

Reply:
We could go with that.

Reply:
@Pakrym I don't understand what you mean by passing this.GetType() alone. The Start() method requires a string. Please can you clarify.

Reply:
You can add an overload that takes Type and use to call the overload that takes the string.